outputs.tf

```diff
@@ -28,11 +28,6 @@ output "cluster_security_group_id" {
   value       = "${local.cluster_security_group_id}"
 }
 
-output "config_map_aws_auth" {
```

Comment:
Can we just keep this as an output? I'm sure someone will still want it. e.g. so they can have the file in their version control or something.

Reply:
I added the configmap as output, but it is a terraform resource now. Which is not the same as the rendered yaml. But I can't think of an easy way to achieve that